scrapy/mail.py

```diff
@@ -98,6 +111,10 @@ def send(self, to, subject, body, cc=None, attachs=(), mimetype='text/plain', ch
                           'mailattachs': len(attachs)})
             return
 
+        rcpts = to[:]
+        if cc:
+            rcpts.extend(cc)
```

Comment:
cc is no longer passed through `list(arg_to_iter(cc))`, and `msg['Cc']` is no longer updated, this looks backwards incompatible

Reply:
Fixed. I noticed that we don't have a `bcc` field as well. Maybe it would be interesting to include this new argument, what do you think?